CHANGELOG.md

```diff
@@ -37,6 +37,9 @@
 * Add Key Color Palettes [#13497](https://github.com/mixxxdj/mixxx/pull/13497)
 * Add overview column with small waveform [#14140](https://github.com/mixxxdj/mixxx/pull/14140)
 * Add a rebindable keyboard shortcut for editing items as a replacement for F2 [#13148](https://github.com/mixxxdj/mixxx/pull/13148)
+* DeveloperTools: restore searchbar shortcut Ctrl+F
+  [#14761](https://github.com/mixxxdj/mixxx/pull/14761)
+  [#14734](https://github.com/mixxxdj/mixxx/issues/14734)
```

Comment:
```suggestion
```
I mentioned the wrong PR in #14761 (fixed, should be PR #13200) and the script picked it up